conf/variant/common/bblayers-qtauto.conf

```diff
@@ -1,7 +1,8 @@
 
 BBLAYERS += "\
   ${BSPDIR}/sources/meta-qt5     \
-  ${BSPDIR}/sources/meta-boot2qt \
+  ${BSPDIR}/sources/meta-boot2qt/meta-boot2qt-distro \
```

Comment:
Cool stuff! I am wondering what is in this distro meta layer of boot2qt?

Reply:
it contains all the additional recipes. At the moment this is only needed because of the packagegroups. Either we stop using them and create our own or ask tqc to make them more generic and move them to the meta-boot2qt layer

Reply:
@Gagi2k thanks for the comment! Understood, maybe we can investigate this later on, it would be great to avoid any distro specific appendments from boot2qt so this kinda structuring is cool from their side. We can pick this up later!